frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.tsx

```diff
@@ -68,26 +68,36 @@ const ParameterLinkedFilters = ({
     [],
   );
 
+  let disableReason = null;
```

Comment:
Could you extract this to a function to avoid mutable variables?

Reply:
Done 96c11e3df45e6d9d2d53fe615c70c190d6fe6e6b